ctapipe/io/datawriter.py

```diff
@@ -46,8 +46,9 @@ def _get_tel_index(event, tel_id):
 #   (meaning readers need to update scripts)
 # - increase the minor number if new columns or datasets are added
 # - increase the patch number if there is a small bugfix to the model.
-DATA_MODEL_VERSION = "v4.0.0"
+DATA_MODEL_VERSION = "v5.0.0"
 DATA_MODEL_CHANGE_HISTORY = """
+- v5.0.0: - Change DL2 telescope-wise container prefixes from {algorithm}_tel to {algorithm}_tel_{kind}
```

Comment:
This indicates a much larger change than has actually happened.

We just renamed `tel_distance` to `tel_impact_distance`. I.e. whe changed the prefix of a single container from '' to 'impact'.

Nothing to do with "algorithms" or "kinds".

Reply:
Well yes, but in https://github.com/cta-observatory/ctapipe/blob/6cc4f66f6e61a8da2bbbc72507d66381adde91e0/ctapipe/io/hdf5eventsource.py#L449 I made the changes as indicated in the history. This currently only adds the `_impact_` to tel_distance, but (in future?) other DL2 telescope-wise parameters will have the "kind", in addition to the "algorithm", with the "tel" in between. I wanted to account for this in the history. 

But I understand, currently this changes only the `tel_distance`, and am fine with making that clearer

Reply:
Sounds good